### PR TITLE
roles and team update

### PR DESCRIPTION
**Pull request by JenTress**
To reflect critical needs based on team priorities.

### CODE REVIEW

File: pages/roles-and-teams.md [hunk 1]
```diff
@@ -2,60 +2,26 @@
...
-### Comms Team
+**Current needs**
 
-**Director of Outreach**
-
-The Director of Outreach drives a comprehensive internal and external communications strategy for 18F. This includes editorial and content on our website, social media, events/conferences, messaging, and general community evangelism.
+**Business Strategists - Digital Transformation**
 
-### Consulting Team
+Business strategists help the agencies deliver on their mission through insights that guide strategy and solve problems for end users. They use workshops and human centered design principles as a means to collaborating with partner agencies, and getting results.
```
Comment (fureigh) [inline pages/roles-and-teams.md:11]: 'human centered' --> 'human-centered'

File: pages/roles-and-teams.md [hunk 2]
```diff
@@ -2,60 +2,26 @@
...
+**Business Strategists - Digital Transformation**
 
-### Consulting Team
+Business strategists help the agencies deliver on their mission through insights that guide strategy and solve problems for end users. They use workshops and human centered design principles as a means to collaborating with partner agencies, and getting results. 
 
-**Acquisition Specialist**
+**Director of Outreach**
 
-Acquisition specialists help teams support digital services through procurement efforts, investigate other agencies’ tools and processes to find opportunities to improve results, and work with contracting officers to help procure goods and services for 18F and 18F partner agencies.
+The Director of Outreach drives a comprehensive internal and external communications strategy for 18F. This includes editorial and content on our website, social media, events/conferences, messaging, and general community evangelism.
```
Comment (fureigh) [inline pages/roles-and-teams.md:15]: Suggested change: 'messaging, and general community evangelism' to 'and messaging, as well as general community evangelism.'

File: pages/roles-and-teams.md [hunk 3]
```diff
@@ -2,60 +2,26 @@
...
 
 **Data Architect**
```
Comment (fureigh) [inline pages/roles-and-teams.md:17]: Let's pluralize this for consistency.

**Comment by pixelwake** [comment]
Closing this PR, this content and other updates in #183

**Pull request closed without merging**